server/util/status/status.go

```diff
@@ -219,3 +220,19 @@ func Message(err error) string {
 	}
 	return err.Error()
 }
+
+// FromContextError converts ctx.Err() to the equivalent gRPC status error from
+// this package.
+func FromContextError(ctx context.Context) error {
+	err := ctx.Err()
```

Comment:
The grpc status library seems to already have this implemented, why don't we call status.FromContextError() here?

Reply:
Done, thanks